.github/workflows/promptflow-sdk-pfs-e2e-test.yml

```diff
@@ -0,0 +1,113 @@
+name: promptflow-sdk-pfs-e2e-test
+on:
+  schedule:
+    - cron: "40 18 * * *" # Every day starting at 2:40 BJT
+  pull_request:
+    branches: [ main ]
+    paths:
+      - src/promptflow/**
+      - scripts/**
+      - '**promptflow-sdk-pfs-e2e-test.yml'
+  workflow_dispatch:
+env:
+  packageSetupType: promptflow_with_extra
+  testWorkingDirectory: ${{ github.workspace }}/src/promptflow
+  PYTHONPATH: ${{ github.workspace }}/src/promptflow
+  IS_IN_CI_PIPELINE: "true"
+jobs:
+  authorize:
+    environment:
+      # forked prs from pull_request_target will be run in external environment, domain prs will be run in internal environment
+      ${{ github.event_name == 'pull_request_target' &&
```

Comment:
always internal here since there is no pull request target.

Reply:
add pull_request_target above, so no need to change here.